### PR TITLE
BN attributes quantization tests

### PR DESCRIPTION
**Pull request by ofirgo**
## Pull Request Description:
Adding tests that enable Batchnorm layer attributes quantization to validate attributes quantization feature.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [x] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tests/keras_tests/feature_networks_tests/feature_networks/bn_attributes_quantization_test.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        f_gamma = [w for w in f_bn_weights if GAMMA in w.name]
+        self.unit_test.assertTrue(len(f_gamma) == 1, "Expecting float model BN layer to have a GAMMA attribute")
+        f_gamma = f_gamma[0]
+        q_gamma = q_bn_weights.get(GAMMA)
+        self.unit_test.assertTrue(q_gamma is not None, "Expecting quantized model BN layer to have a GAMMA attribute")
+        self.unit_test.assertTrue(np.any(f_gamma != q_gamma),
+                                  "Float and quantized GAMMA attributes are expected to have different values")
+
+        # Verify output difference (sanity)
+        y = float_model.predict(input_x)
```
Comment (reuvenperetz) [inline tests/keras_tests/feature_networks_tests/feature_networks/bn_attributes_quantization_test.py:165]: I think the predictions part is not needed (and opens a door for randomness issues).

File: tests/pytorch_tests/model_tests/feature_models/bn_attributes_quantization_test.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                                  "Float and quantized GAMMA attributes are expected to have different values")
+
+        # Verify output difference (sanity)
+        in_torch_tensor = to_torch_tensor(input_x[0])
+        set_model(float_model)
+        y = float_model(in_torch_tensor)
+        y_hat = quantized_model(in_torch_tensor)
+        self.unit_test.assertTrue(y.shape == y_hat.shape, msg=f'Out shape is not as expected!')
+        cs = cosine_similarity(torch_tensor_to_numpy(y), torch_tensor_to_numpy(y_hat))
+        self.unit_test.assertTrue(np.isclose(cs, 1, atol=1e-3), msg=f'Fail cosine similarity check:{cs}')
```
Comment (reuvenperetz) [inline tests/pytorch_tests/model_tests/feature_models/bn_attributes_quantization_test.py:185]: Same

**Review by reuvenperetz — APPROVED** [review]
